tests/timeseries/test_sort_timestamps_monotonically.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import pandas as pd
+from random import randint
+from janitor.timeseries import sort_timestamps_monotonically
+
+
+# Random data for testing
+@pytest.fixture
+def timeseries_dataframe() -> pd.DataFrame:
+    """
+    Returns a time series dataframe
+    """
+    ts_index = pd.date_range("1/1/2019", periods=1000, freq="1H")
+    v1 = [randint(1, 2000) for i in range(1000)]
+    test_df = pd.DataFrame({"v1": v1}, index=ts_index)
+    return test_df
+
+
+@pytest.mark.timeseries
+def test_sort_timestamps_monotonically(timeseries_dataframe):
```

Comment:
Can this test be split out please? Yes, there's a bit more boilerplate involved, but if and when a test fails, having them split out makes it easier to identify which exact test failed. Basically, each assert would go to its own function. 

Some proposed function names:

- `def test_sort_timestamps_monotonically`
- `def test_sort_timestamps_monotonically_decreasing`
- `def test_sort_timestamps_monotonically_strict`